versions.tf

```diff
@@ -1,4 +1,9 @@
 terraform {
+  provider_meta "equinix" {
+    # TEMPLATE: Set the name of the module below 
+    module_name = "template"
```

Comment:
The `module_name` should contain the name of this module (in this case, `equinix-kubernetes-cluster`).